ios/PhyWeb/UI/PWBeaconsViewController.m

```diff
@@ -329,6 +329,21 @@ - (void)_updateBeaconsNow {
 - (void)_sort {
   BOOL stableMode = [[PWBeaconManager sharedManager] isStableMode];
 
+  BOOL hasRank = NO;
+  for(PWBeacon * beacon in _beacons) {
+    if ([beacon hasRank]) {
+      hasRank = YES;
```

Comment:
Fair -- though you still want to set rank to 1000 for those, no?  So we could just say "if no rank then 1000" for all values without checking for server side support?